spec/puppetserver/ca/action/clean_spec.rb

```diff
@@ -12,6 +12,12 @@
 
   subject { Puppetserver::Ca::Action::Clean.new(logger) }
 
+  def without_possible_deprecation(io)
+    io.each_line.reject do |line|
+      line =~ /Found existing CA directory at the old location/
+    end
+  end
+
```

Comment:
yeah, the tests in the file want to test that certain things they do _don't_ cause warnings/errors, so for those assumptions to still be valid we want to remove potential warnings from the cadir location.

Reply:
Open to other ideas for names? `without_possible_cadir_deprecation`? `maybe_remove_cadir_deprecation`? We could remove `maybe` or `possible` to be more terse, though if the old cadir exists and warns during these tests depends on the user running the tests, iirc.

Reply:
ahhh okay, so the tests are saying "this doesn't cause warnings/errors", but they WILL cause the deprecation warning, so we just strip those out because we don't care about them? okay. Can we call this something like `remove_deprecation_warning` or something like that? I think that would make the intention clearer to me.

then my question is, why is the deprecation warning going to stderr? Is that just what warnings do by default?

Reply:
what did you think of `without_cadir_deprecation`?

And, yeah, warnings and errors go to stderr.

Reply:
I like using a verb so that it's clear what the intention is. `remove_cadir_deprecation`?